typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
 
 
 # get the directory path to this file; 
-LIBS_DIR = os.path.dirname(os.path.abspath(__file__))
-PLUGIN_DIR = os.path.dirname(LIBS_DIR)
-PACKAGES_DIR = os.path.dirname(PLUGIN_DIR)
+LIBS_DIR = dirname(dirname(os.path.abspath(__file__)))
```

Comment:
This is confusing, as you have a 'libs' dir (i.e. that this file is in), yet this folder points to '/Users/bill/src/TypeScript-Sublime-Plugin/typescript' on my machine (i.e. the package dir).  Is this just named confusingly?